src/stylesheets/app.scss

```diff
@@ -0,0 +1,276 @@
+@import "base/typography";
+@import "base/vars";
+
+body{
+  margin: 0;
+  background-color: #d2e7e2; // only temporary
+}
+ul{
+  padding: 0;
+}
+sup{
+  color: #e95e67;
+  text-decoration: underline;
+}
+
+/*** Top Fold styles ***/
+#top-fold{
+  background: #d2e7e2;
+  padding-top: 50px;
+  overflow: hidden;
+  padding-bottom: 15%; //only temporary
+
+  #title-box{
+    position: relative;
+  }
+  #title{
+    margin: auto;
+    max-width: 50%;
+    @media (max-width: 600px){
+      max-width: 85%;
+      margin-bottom: 8%;
+    }
+  }
+
+  #subtitle{
+    color: #0e5254;
+    font-family: 'Myriad Pro';
+    font-size: 14px;
+    letter-spacing: 3px;
+    margin-bottom: 2%;
+    text-align: center;
+    text-transform: uppercase;
+  }
+
+  #yellow-pill{
+    left: 22%;
+    top: -30%;
+    width: 7%;
+    position: absolute;
+    transform: rotate(
+      35deg
+    );
+    @media (max-width: 600px){
+      width: 20%;
+      top: -55%;
+      left: 1%;
+      transform: rotate(
+        40deg
+      );
+    }
+  }
+
+  #pills{
+    left: 12%;
+    position: absolute;
+    top: 65%;
+    width: 10%;
+    @media (max-width: 600px){
+      display: none;
+    }
+  }
+
+  #more-pills{
+    position: absolute;
+    top: -20%;
+    right: 12%;
+    width: 20%;
+    transform: rotate(10deg);
+    @media (max-width: 600px){
+      width: 60%;
+      top: -40%;
+      right: -35%;
+    }
+  }
+
+  #purple-pills{
+    position: absolute;
+    right: 10%;
+    width: 12%;
+    @media (max-width: 600px){
+      width: 35%;
+      bottom: -30%;
+      right: -22%;
+      z-index: 4;
+    }
+  }
+}
+
+/***** Pillbar styles *****/
+
+.pillbar{
+  color: #d2e7e2;
+  background-image: linear-gradient(0deg, #e95e67 75%, #194e67 25%);
+  background-image: -webkit-linear-gradient(0deg, #e95e67 75%, #194e67 25%);
+  -moz-box-shadow: inset 2px -15px 2px rgba(0, 0, 0, 0.1);
+  -webkit-box-shadow: inset 2px -15px 2px rgba(0, 0, 0, 0.1);
+  box-shadow: inset 2px -15px 2px rgba(0, 0, 0, 0.1);
+  height: 35px;
+  border-radius: 70px;
```

Comment:
looks like a bunch of these properties are out of alphabetical order here and in other places. we use alphabetized CSS properties to prevent duplication. depending on your editor, you should be able to select all the properties and hit `F5` to alphabetize them for you